product_template_tags_group/security/product_template_tag_group_rule.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+  ~ Copyright 2022 Simone Rubino - TAKOBI
+  ~ License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+  -->
+
+<odoo noupdate="1">
+    <record id="product_template_tag_group_rule_company" model="ir.rule">
+        <field name="name">Multi-company for Product Tag Group</field>
+        <field name="model_id" ref="model_product_template_tag_group"/>
+        <field name="perm_read" eval="1"/>
+        <field name="perm_create" eval="1"/>
+        <field name="perm_write" eval="1"/>
+        <field name="perm_unlink" eval="1"/>
+        <field name="domain_force">['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
+    </record>
+</odoo>
```

Comment:
I think a multi company rule is not need, as the main data are products, which have multi company rules.
I would follow the same approach as `product.category` about this